docs/getting-started-guide/master.adoc

```diff
@@ -9,29 +9,29 @@ include::topics/templates/document-attributes.adoc[]
 :getting-started-guide:
 
 :_content-type: ASSEMBLY
-= Introduction to the {ProductName}
+= Getting started with the {ProductName}
 
 //Inclusive language statement
 include::topics/making-open-source-more-inclusive.adoc[]
 
-== Introduction
+// == Introduction
 
 // About the {IntroToMTABookName}
-include::topics/mta-about-the-intro-to-mta-guide.adoc[leveloffset=+2]
+include::topics/mta-about-the-intro-to-mta-guide.adoc[leveloffset=+1]
 
 // About the Toolkit
-include::topics/mta-what-is-the-toolkit.adoc[leveloffset=+1]
+// Info included in the module above ^ include::topics/mta-what-is-the-toolkit.adoc[leveloffset=+1]
 
 // Windup Features
 include::topics/new-mta-features.adoc[leveloffset=+2]
 
 // Windup Rules
 include::topics/about-rules.adoc[leveloffset=+2]
 
-== Supported configurations
+//== Supported configurations
 
 // Supported Migration Paths
-include::topics/migration-paths.adoc[leveloffset=+2]
+include::topics/migration-paths.adoc[leveloffset=+1]
 
 // Prerequisites
 == Prerequisites
```

Comment:
This module is somewhat confusing. Can we at least say what are these prerequisites for? MTA installation? Needs clarification.

Reply:
Grand so, my only concern here is the errors in Pantheon, which happened last time a writer tried to change the name

If we can agree to change the name for 7.1 when i build this release in Pantheon then we should be fine

But changing it now might cause some problems.

If you are really determined to change it now I will not stop you... but let us not change before Tuesday as I need to publish Release Notes for 7.0.2

Reply:
Ok, got it, Andy! Let's then wait until Tuesday when you're done with Release Notes.

Reply:
Cool... grand with me, you can make an absolute mess of the repo so it never works again but only AFTER Tuesday. Just be prepared to take the blame and me to say "Radhika! Masha done broke Pantheon and ran away"  :rofl: